EPMLABSBRN-22 changes in database structure and dtos

goopnigoop opened the pull request:
- fixing database issues and relations
- flow to get tasks
- initial data for database

goopnigoop committed:
EPMLABSBRN-22 changes in database structure and dtos

- fixing database issues and relations
- flow to get tasks
- initial data for database

rpcep commented on src/main/kotlin/com/epam/brn/service/UserDetailsService.kt:
```diff
@@ -12,15 +12,15 @@ class UserDetailsService(@Autowired val userDetailsDAO: UserDetailsRepository) {
-        log.info("add new user $name $email $phone")
-        val newUser = userDetailsDAO.save(UserDetails(1, name, email, phone))
-        return newUser.id
+//        log.info("add new user $name $email $phone")
```
I would remove all commented code and just leave a //TODO.

goopnigoop replied:
totally agree

goopnigoop committed:
EPMLABSBRN-22 changes in database structure and dtos

- fixing database issues and relations
- flow to get tasks
- initial data for database

goopnigoop committed:
EPMLABSBRN-22 removal of unnecessary annotations

RiveriRiv commented on src/main/kotlin/com/epam/brn/model/Exercise.kt:
```diff
@@ -1,21 +1,67 @@
+        tasks = tasks.map { task -> task.toDto() }.toMutableSet()
+    )
+
+    override fun equals(other: Any?): Boolean {
```
What do you think about using somthing like lombok annotation @EqualsAndHashcode for generating this methods?
